app/services/idv/steps/document_capture_step.rb

```diff
@@ -8,11 +8,7 @@ class DocumentCaptureStep < DocAuthBaseStep
       ].freeze
 
       def call
-        if request_should_use_stored_result?
-          handle_stored_result
-        else
-          post_images_and_handle_result
-        end
+        handle_stored_result if !FeatureManagement.document_capture_async_uploads_enabled?
```

Comment:
Accidentally stumbled into a fun little bug! When async uploads are enabled, we don't want to (and can't) handle the stored result here. The [handling is done in `VerifyDocumentStatusAction`](https://github.com/18F/identity-idp/blob/72fa0e270b58a097ce5efa6c3e24f75d0d757634/app/services/idv/actions/verify_document_status_action.rb#L53-L59), and the stored result is _deleted_, so `request_should_use_stored_result?` would always be false. This meant that with async uploads enabled, a JS-enabled submission would still go through the `post_images_and_handle_result` procedure below. This would result in a failure due to absent values, but because the client-side communicates with the `VerifyDocumentStatusAction` class, it would still proceed to the next step. But the error generated by this step would manifest in a "Please fill in this field" message on the next screen (SSN step).

<img width="308" alt="image" src="https://user-images.githubusercontent.com/1779930/170713487-6b5783be-2366-4a32-bbe6-cb6884623c7d.png">

(cc @anniehirshman-gsa)